.github/workflows/CICD.yml

```diff
@@ -235,8 +247,6 @@ jobs:
           # { os, target, cargo-options, features, use-cross, toolchain }
           - { os: ubuntu-latest  , target: arm-unknown-linux-gnueabihf , features: feat_os_unix_gnueabihf , use-cross: use-cross }
           - { os: ubuntu-latest  , target: aarch64-unknown-linux-gnu   , features: feat_os_unix_gnueabihf , use-cross: use-cross }
-          - { os: ubuntu-latest  , target: x86_64-unknown-linux-gnu    , features: feat_os_unix           , use-cross: use-cross }
```

Comment:
@rivy I'm missing ubuntu-latest x86_64. Is there a specific reason why you removed it or was it an oversight?

Reply:
Looks like a mistake in 298851096ef94. I only meant to remove the single line with the deprecated `ubuntu-16.04` image.